### ISSUE TITLE
List of user who thanked the post

### ISSUE THREAD

**Issue by kodyazari** [rxu/thanks_for_posts#120]
Hi,
An additional page can be created. All thankers to the post are listed on this page.
A better version of this could be like this:
Popup box opened with ajax when we click on the number of likes. There may be more users thanking. "Load more" link via ajax under popup box.
![xzcv](https://github.com/rxu/thanks_for_posts/assets/167031252/61fbeaa5-e773-40be-ac87-3d1207129864)

**Comment by eunaotenhoid** [comment]
Yes, and being able to see the other users who thanked me, in addition to the 250, would be great too.
